src/formats/CML.cpp

```diff
@@ -100,9 +100,9 @@ static void read_property_(T& container, pugi::xml_node& node, const std::string
     std::string data_type_str = "xsd:string";
     auto data_type = node.attribute("dataType");
     if (!data_type) {
-        warning("[CML] {} has no data type, assuming string.", title);
+        warning("{} has no data type, assuming string", title);
```

Comment:
Same comment as above. In fact, it's even more important for warnings because in this case, the program continues and this could lead to odd results downstream.